project/dexter-metrics/src/test/com/samsung/sec/dexter/metrics/util/MetricUtilTest.java

```diff
@@ -28,5 +35,80 @@ public void test() {
 		
 	//MetricUtil.getFunctionLOCArray(source, 0, 14);
 	}
-
+	
+	@Test
+	public void MetricUtilTest_NonexistantFile() {
+		MetricUtil tester = new MetricUtil();
+		int start = 0;
+		int end = 10;
+		DexterRuntimeException exception = null;
+		try {
+			tester.getFunctionLOCArray(".\\ThereIsNoSuchFile.nope", start, end);
+		}
+		catch(DexterRuntimeException e) {
+			exception=e;
+			if(exception==null) {
+				Assert.fail();
+			}
+		}
+	}
+	
+	@Test
+	public void MetricUtilTest_EmptyFilePath() {
+		MetricUtil tester = new MetricUtil();
+		int start = 0;
+		int end = 10;
+		tester.getFunctionLOCArray("", start, end);
+	}
+	
+	@Test
+	public void MetricUtilTest_StartGreaterThanEnd() {
+		MetricUtil tester = new MetricUtil();
+		int start = 4;
+		int end = 2;
+		tester.getFunctionLOCArray(TestFilePath1, start, end);
+	}
+	
+	@Test
+	public void MetricUtilTest_StartNegative() {
+		MetricUtil tester = new MetricUtil();
+		int start = -10;
+		int end = 2;
+		tester.getFunctionLOCArray(TestFilePath1, start, end);
+	}
+	
+	@Test
+	public void MetricUtilTest_TooLargeEnd() {
```

Comment:
The test case fails. Assert functions must be included to ensure that an Exception occurs.